extras.json

```diff
@@ -33,8 +33,11 @@
   "sky": {
     "strength": 6,
     "color": [0.854, 0.862, 1],
+    "ambientLight": {
```

Comment:
ambientLight and ambientOcclusion are separate objects now.

Reply:
Yes, fixed that, thanks